### PR TITLE
Use -k for each arg in duplicates plugin

### PR DESCRIPTION
**Pull request by jrobeson**
This was extracted from the click branch. I'm not sure how we could make this BC, but if you have any ideas, then let me know.

Also, please suggest any wording changes.

### CODE REVIEW

File: docs/plugins/duplicates.rst [hunk 1]
```diff
@@ -46,8 +46,8 @@ file. The available options mirror the command-line options:
 - **checksum**: Use an arbitrary command to compute a checksum
   of items. This overrides the ``keys`` option the first time it is run;
   however, because it caches the resulting checksum as ``flexattrs`` in the
-  database, you can use ``--keys=name_of_the_checksumming_program
-  any_other_keys`` (or set configuration ``keys`` option) the second time
+  database, you can use ``--key=name_of_the_checksumming_program
+  --key=any_other_keys`` (or set configuration ``keys`` option) the second time
```
Comment (sampsyo) [inline docs/plugins/duplicates.rst]: This isn't your fault, but the parenthetical should probably read "or set the keys configuration option".

**Comment by sampsyo** [comment]
I like this change; it not only makes the code simpler but also makes the interface more predictable in my opinion. The only concern, as you noted, is BC—and I'm inclined to say that we should just go ahead with the change, since I can't imagine there are too many scripts out there relying on the old behavior.

File: docs/plugins/duplicates.rst (hunk 1 above)
Comment (jrobeson) [inline docs/plugins/duplicates.rst]: so other than this, it's good to go?

File: docs/changelog.rst [hunk 2]
```diff
@@ -18,6 +18,12 @@ to know:
 * Please don't package beets as a Python 3 application *yet*, even though some
   things may appear to work under Python 3.4 and later.
 
+UI Change:
```
Comment (jrobeson) [inline docs/changelog.rst]: this header doesn't match up with the others. Other headers are sentences.

**Comment by jrobeson** [comment]
I hope you like it, as you're the author :)

File: docs/plugins/duplicates.rst (hunk 1 above)
Comment (sampsyo) [inline docs/plugins/duplicates.rst]: Yep!

File: docs/changelog.rst (hunk 2 above)
Comment (sampsyo) [inline docs/changelog.rst]: Good point. I usually massage the release notes quite a bit before doing the final release anyway, so don't worry about it too much.

**Comment by sampsyo** [comment]
Ha! I didn't mean to give myself a compliment. I just have a terrible memory, apparently...

File: docs/plugins/duplicates.rst (hunk 1 above)
Comment (jrobeson) [inline docs/plugins/duplicates.rst]: i'm going to change it after i merge it, as it's unrelated to the PR.

Comment (sampsyo) [inline docs/plugins/duplicates.rst]: Good idea. Thanks for cleaning up someone else's writing.

**Pull request merged**